shell/artifacts/Common/List.manifest

```diff
@@ -6,27 +6,38 @@
 // subject to an additional IP rights grant found at
 // http://polymer.github.io/PATENTS.txt
 
+shape HostedParticleShape
+  HostedParticleShape(in ~a)
+  consume item
+
+particle Multiplexer in 'source/Multiplexer.js'
```

Comment:
there also is Multiplexer manifest here: https://github.com/PolymerLabs/arcs/blob/master/shell/artifacts/Common/Multiplexer.manifest
but that one consumes "annotation", so i assume this is why both are needed?

Reply:
Yes.